data/Symbeeosis/codedLists.ttl

```diff
@@ -0,0 +1,55 @@
+bdg:AssayCompound a rdfs:Class; rdfs:subClassOf skos:Concept;
+  rdfs:label "Assay compound codelist class".
+<assayCompound> a skos:ConceptScheme;
+  rdfs:label "Assay compound codelist scheme".
+<assayCompound/DPPH> a skos:Concept, bdg:AssayCompound;
+  skos:inScheme      <assayCompound>; skos:topConceptOf <assayCompound>;
+  skos:prefLabel     "DPPH" ;
+  skos:exactMatch       wd:Q5205953 .
```

Comment:
Better use PubChem or CHEBI IDs, eg DPPH is
- https://pubchem.ncbi.nlm.nih.gov/compound/74358: rich html page
- http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID74358: RDF.
```ttl
@prefix pubchem: <http://rdf.ncbi.nlm.nih.gov/pubchem/compound/CID>.
```